### ISSUE TITLE
When removing kernel bundles all configurations are not removed

### ISSUE THREAD

**Issue by LibertadCruz** [clearlinux/distribution#886]
When you have installed more than one kernel, and you can boot from the different kernels. Once you remove the kernel bundle it apperantly shows the bundle no longer exists. But I can still see the different kernels configuration files in the /EFI/org.clearlinux path.  Plus it still shows them as booting options in the initial menu because they still exists as bootloader entries.  CL version 29880

**Comment by ahkok** [comment]
This is normal. Clear-boot-manager will prune them if there are too many, but these files are not like package content - there are very simple reasons why you can't just remove kernel files when a bundle is removed - it might leave you unable to boot your system.

**Comment by fenrus75** [comment]
I wonder if we do not include the initrd stuff in the garbage collection
algorithm though

**Comment by miguelinux** [comment]
> This is normal. Clear-boot-manager will prune them if there are too many, but these files are not like package content - there are very simple reasons why you can't just remove kernel files when a bundle is removed - it might leave you unable to boot your system.
> 

Should we add an option to remove kernel to CBM?

or create a documentation on how to remove the kernel from ESP?

**Comment by ahkok** [comment]
Yes, I didn't close the ticket for a reason - I've been looking into doing some changes to CBM myself since I ran into many unfinished ends already, and we should put this on top of that work.
